docs/documentation/features/providedServices.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: documentation
+---
+
+# Provided Services
+
+In this chapter useful services of the Eclipse SmartHome project are described. 
+
+## Obtaining the default IP address
```

Comment:
Use the service name for the header: "Network Address Service" or something like that.